guides/v2.1/howdoi/checkout/checkout_zip.md

```diff
@@ -17,6 +17,28 @@ When a shopper specifies the country and ZIP code in the shipping address during
 In Magento the input masks for the **ZIP code** field are specified in the `<Magento_Directory_module_dir>/etc/zip_codes.xml`. Input masks are specified per country, and are entered in the form of regular expressions.
 The syntax of defined by the [zip_code.xsd]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Directory/etc/zip_codes.xsd) scheme.
 
+The following table defines the `zip` node attributes:
+
+ Attribute name | Required | Description
+--- | --- | ---
+`countryCode` | Yes | The country code (Alpha-2 format) for which the zip is defined
+
```

Comment:
Please, provide also an example of the `zip` node. So in result we have an example for the `zip` node and an example for the `codes` node.
Thank you

Reply:
Hi @rogyar a `zip` node example has been provided.
Thank you.